packages/ringcentral-integration/modules/ConversationLogger/index.js

```diff
@@ -80,14 +81,17 @@ export default class ConversationLogger extends LoggerBase {
     this._tabManager = tabManager;
     this._isLoggedContact = isLoggedContact;
     this._formatDateTime = formatDateTime;
+    this._isAutoUpdate = isAutoUpdate;
     this._storageKey = `${this._name}Data`;
     this._storage.registerReducer({
       key: this._storageKey,
       reducer: getDataReducer(this.actionTypes),
     });
 
     this.addSelector('conversationLogMap',
-      () => this._messageStore.messages,
+      () => Object
+        .values(this._messageStore.conversationStore)
```

Comment:
This  will create new object every time. Please move this into reselect function.